techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md

```diff
@@ -1,8 +1,8 @@
 # Not Using a Proxy Pass
 
-> ### `Important`
-> This package is set up to use a proxy pass to the backend by default using `/api`.  
-> This means that all traffic to `<frontend url>/api` will be directed to `<backend_url>`.  
+!!! important "Important"
```

Comment:
This just makes it look like the "note" so you  can change it back

Reply:
@BradyMitch what do you think of the "warning" blocks for the Information sections? 
Example:
![warn_test](https://github.com/bcgov/citz-imb-sso-react/assets/78506153/923a0c45-07ee-4874-bc73-2254a5c912b4)

Reply:
I think both info and warning look good. I think maybe warning stands out more.